client/dashboard/sites/overview/agency-site-share-card.tsx

```diff
@@ -0,0 +1,17 @@
+import { __ } from '@wordpress/i18n';
+import { link } from '@wordpress/icons';
+import OverviewCard from '../overview-card';
+import type { Site } from '../../data/types';
+
+export default function AgencySiteShareCard( { site }: { site: Site } ) {
+	return (
+		<OverviewCard
+			icon={ link }
+			heading={ __( 'Preview link enabled' ) }
```

Comment:
Shouldn't this say `Preview link disabled` if the share link is not enabled? It feels weird 🤔 

|<img width="344" height="162" alt="Image" src="https://github.com/user-attachments/assets/15105fc0-83d7-43d6-8515-ae85a5273cf1" />|<img width="464" height="150" alt="Image" src="https://github.com/user-attachments/assets/e6361d45-4ac2-498e-90fe-30ad5aa030b2" />|
|-|-|

Reply:
FWIW, we also use "preview link" when the "Enable share link" toggle is updated.

<img width="237" height="56" alt="SCR-20250717-kguu" src="https://github.com/user-attachments/assets/01c8a2dd-d2b4-44fa-a013-a144eb05c719" />

Reply:
Updated in 3cac015c9f0a84d063a58f8d8cecb64ee495c8fa.